packages/protocol/releaseData/nonstandarVersionReports/baklava_fix_report_report.json

```diff
@@ -0,0 +1,859 @@
+{
+  "oldArtifactsFolder": "/Users/martinvol/celo/celo-monorepo/packages/protocol/build/core-contracts.v3/contracts",
+  "newArtifactsFolder": "/Users/martinvol/celo/celo-monorepo/packages/protocol/build/core-contracts.v8/contracts",
+  "exclude": "/.*Test|Mock.*|I[A-Z].*|.*Proxy|MultiSig.*|ReleaseGold|SlasherUtil|UsingPrecompiles/",
+  "report": {
+    "contracts": {
+      "Escrow": {
+        "changes": {
+          "storage": [],
+          "major": [
+            {
+              "contract": "Escrow",
+              "signature": "setRegistry(address)",
+              "type": "MethodRemoved"
+            },
+            {
+              "contract": "Escrow",
+              "signature": "initialize(address)",
+              "type": "MethodRemoved"
+            }
+          ],
+          "minor": [
+            {
+              "contract": "Escrow",
+              "signature": "transferWithTrustedIssuers(bytes32,address,uint256,uint256,address,uint256,address[])",
+              "type": "MethodAdded"
+            },
+            {
+              "contract": "Escrow",
+              "signature": "getTrustedIssuersPerPayment(address)",
+              "type": "MethodAdded"
+            },
+            {
+              "contract": "Escrow",
+              "signature": "getDefaultTrustedIssuers()",
+              "type": "MethodAdded"
+            },
+            {
+              "contract": "Escrow",
+              "signature": "MAX_TRUSTED_ISSUERS_PER_PAYMENT()",
+              "type": "MethodAdded"
+            },
+            {
+              "contract": "Escrow",
+              "signature": "addDefaultTrustedIssuer(address)",
+              "type": "MethodAdded"
+            },
+            {
+              "contract": "Escrow",
+              "signature": "removeDefaultTrustedIssuer(address,uint256)",
+              "type": "MethodAdded"
+            },
+            {
+              "contract": "Escrow",
+              "signature": "initialize()",
+              "type": "MethodAdded"
+            }
+          ],
+          "patch": [
+            {
+              "contract": "Escrow",
+              "type": "DeployedBytecode"
+            },
+            {
+              "contract": "Escrow",
+              "dependency": "Signatures",
+              "type": "LibraryLinkingChange"
+            }
+          ]
+        },
+        "versionDelta": {
+          "storage": "=",
+          "major": "+3",
```

Comment:
On `core-contracts.v8`, Escrow appears to have version 1.2.0.0, why the +3 major increase?